docs/4.3. Behaviour - Tokens.md

```diff
@@ -96,39 +96,66 @@ _Contains information particular to the NMOS API the token is intended for_
 
 The `x-nmos-api` claim MUST be included in the token. The value of the claim is a JSON object, the contents of which
 are `key:value` pairs, with the `key` identifying an individual AMWA NMOS API using the identifier found in the DNS-SD
-advertisement, and the `value` being a JSON array indicating access permissions for that API. An empty `x-nmos-api`
+advertisement, and the `value` being a JSON object indicating access permissions for that API. An empty `x-nmos-api`
 claim MUST indicate no access to any of the APIs. A missing API key MUST indicate no access to that specific API. An
-empty access array MUST indicate no access to that specific API also.
+empty access permissions object MUST indicate no access to that specific API also.
 
 ```json
 "x-nmos-api": {
-  "register": [],
-  "connection": []
+  "register": {},
+  "connection": {}
 }
 ```
 
-The `value` for each NMOS API indicates the access rights of the given user for that specific API. The value required
-by the Resource Server MUST exactly match one of the values in the array. General access rights include:
+##### The Access Permissions Object
+
+The value of each NMOS API key is the access permissions object for the given user for that specific API. The access
+permission object consists of `key:value` pairs in which the `key` denotes a specific access right. General access
+rights include:
 *   **write** - ability to insert a new, or edit an existing, resource. This takes the form of *POST*, *PUT*,
     *PATCH* and *DELETE* HTTP request verbs.
 *   **read** - ability to retrieve the resource. This takes the form of *GET*, *OPTIONS* and *HEAD* HTTP request
     verbs.
 
-If a value of "write" is given in an access array, and the intention is that the given user also has "read" access to
-a given API, the value of "read" MUST also occur in the array.
+If an access right value of "write" is given in the access permission object, and the intention is that the given user
+also has "read" access to the given API, a value of "read" MUST also be present in the access permissions object.
+
+Individual AMWA specifications MAY specify additional access rights specific to the individual API. These MUST be
+registered in the [AMWA NMOS Parameter Registers].
 
-Individual AMWA specifications MAY specify additional values specific to the individual API. These MUST be registered
-in the [AMWA NMOS Parameter Registers].
+The `value` corresponding to each access right is a JSON array containing path specifiers that the user is permitted
+to perform the specific request against. Wildcards are permitted in the path specifiers. The path specifiers MUST use
+dot notation to signify specific sub-paths of the endpoint. For NMOS API's following the standard pattern, all path
```

Comment:
Is there a reason we're not using slashes? Dots would be fine for now, but it feels like there's a danger of a path accidentally using a dot in the future and confusing things.